modules/create-wif-cluster-cli.adoc

```diff
@@ -0,0 +1,196 @@
+2024/09/26 13:05:51 IAM service account openshift-machine-api-gcp-oeub created
+2024/09/26 13:05:51 IAM service account osd-deployer-oeub created
+2024/09/26 13:05:52 IAM service account cloud-credential-operator-oeub created
+2024/09/26 13:05:52 IAM service account openshift-cloud-network-c-oeub created
+2024/09/26 13:05:53 IAM service account openshift-ingress-gcp-oeub created
+2024/09/26 13:05:55 Role "osd_deployer_v4.17" updated
+----
+--
++
+** Create a WIF configuration in manual mode by running the following command:
++
+[source,terminal]
+----
+$ ocm gcp create wif-config --name <wif_name> \ <1>
+  --project <gcp_project_id> \ <2>
+  --manual-mode
+----
+<1> Replace `<wif_name>` with the name of your WIF configuration.
+<2> Replace `<gcp_project_id>` with the ID  of the {GCP} project where the WIF configuration will be implemented.
++
+Once the WIF is configured, the following service accounts, roles, and groups are created.
++
+.WIF configuration service accounts, group and roles
+[cols="2a,3a",options="header"]
+|===
+
+|Service Account/Group
+|GCP pre-defined roles and Red Hat custom roles
+
+
+|osd-deployer
+|osd_deployer_v4.17
+
+
+|osd-control-plane
+|- compute.instanceAdmin
+- compute.networkAdmin
+- compute.securityAdmin
+- compute.storageAdmin
+
+|osd-worker
+|- compute.storageAdmin
+- compute.viewer
+
+|cloud-credential-operator-gcp-ro-creds
+|cloud_credential_operator_gcp_ro_creds_v4.17
+
+|openshift-cloud-network-config-controller-gcp
+|openshift_cloud_network_config_controller_gcp_v4.17
+
+|openshift-gcp-ccm
+|openshift_gcp_ccm_v4.17
+
+|openshift-gcp-pd-csi-driver-operator
+|- compute.storageAdmin
+- iam.serviceAccountUser
+- resourcemanager.tagUser
+- openshift_gcp_pd_csi_driver_operator_v4.17
+
+|openshift-image-registry-gcp
+|openshift_image_registry_gcs_v4.17
+
+|openshift-ingress-gcp
+|openshift_ingress_gcp_v4.17
+
+|openshift-machine-api-gcp
+|openshift_machine_api_gcp_v4.17
+
+|Access via SRE group:sd-sre-platform-gcp-access
+|sre_managed_support
+
+|===
+
+For further details about WIF configuration roles and their assigned permissions, see link:https://github.com/openshift/managed-cluster-config/blob/master/resources/wif/4.17/vanilla.yaml[managed-cluster-config].
+
+[id="create-wif-cluster_{context}"]
+== Creating a WIF cluster
+
+.Procedure
+You can create a WIF cluster using the `interactive` mode or the `non-interactive` mode.
+
+In `interactive` mode, attributes are displayed automatically as prompts during the creation of the cluster, and you specify the values for those prompts based on specified requirements.
+
+In `non-interactive` mode, you specify the values for specific attributes within the command.
+
+* Based on your mode preference, run one of the following commands to create an {product-title} on (GCP) cluster with WIF configuration:
+
+** Create a cluster in interactive mode by running the following command:
++
+[source,terminal]
+----
+ocm create cluster --interactive <1>
```

Comment:
Hi @mletalie, would it be worth following the ROSA CLI guide example here, and having an example output to explain the various CLI fields? Be more work I know. 
![image](https://github.com/user-attachments/assets/904d0b04-6b85-469d-a080-a947ec9a7117)